homework_3/calculator/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/superbet-group/code-cadets-2021/lecture_3/03_project/calculator
+
+go 1.14
+
+require (
+	github.com/kelseyhightower/envconfig v1.4.0
+	github.com/mattn/go-sqlite3 v1.14.7
+	github.com/pkg/errors v0.9.1
+	github.com/streadway/amqp v1.0.0
+	github.com/superbet-group/code-cadets-2021/lecture_3/03_project/controller v0.0.0-20210522111216-09b4d105670d
```

Comment:
You do not need this dependency.
Possibly this is here due to some of the code functionality copy-paste.
Try to run `go mod tidy` if it does not help check where you use this dependency